src/wp-includes/taxonomy.php

```diff
@@ -1559,6 +1559,18 @@ function term_exists( $term, $taxonomy = '', $parent = null ) {
 		$defaults['fields']   = 'all';
 	}
 
+	/**
+	 * Filters default arguments for checking if term exists.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param array      $defaults An array of arguments for term_exists.
+	 * @param int|string $term     The term to check. Accepts term ID, slug, or name.
+	 * @param string     $taxonomy Optional. The taxonomy name to use.
+	 * @param int        $parent   Optional. ID of parent term under which to confine the exists search.
```

Comment:
```suggestion
	 * @param string     $taxonomy The taxonomy name to use. An empty string indicates the search is against all taxonomies.
	 * @param int|null   $parent   ID of parent term under which to confine the exists search. Null indicates the search is unconfined.
```

The variables aren't really optional, WP will always pass them if the developer requests them. Certain values have special meaning.